scripts/pi-hole/lua/header_authenticated.lp

```diff
@@ -13,7 +13,7 @@ mg.include('header.lp','r')
     <script src="<?=pihole.fileversion('scripts/vendor/datatables.min.js')?>"></script>
     <script src="<?=pihole.fileversion('scripts/vendor/datatables.select.min.js')?>"></script>
     <script src="<?=pihole.fileversion('scripts/vendor/datatables.buttons.min.js')?>"></script>
-    <script src="<?=pihole.fileversion('scripts/vendor/chart.umd.min.js')?>"></script>
```

Comment:
Why are we dropping the `min` even if this is obviously a minified file? Is the answer: Because that's how they call the file upstream? (even if it seems strange they do this)